torch/_dynamo/variables/user_defined.py

```diff
@@ -183,7 +183,7 @@ def var_getattr(self, tx: "InstructionTranslator", name: str) -> "VariableTracke
         try:
             obj = inspect.getattr_static(self.value, name)
         except AttributeError:
-            obj = None
+            raise_observed_exception(AttributeError, tx)
```

Comment:
I think we can't do this for metaclasses

![image](https://github.com/user-attachments/assets/a4c66572-4c9e-4562-b76b-e23e4c67515b)